testing/configuration/url_dev.yaml

```
@@ -0,0 +1,2 @@
+page_url:
+  LandingPage: http://13.234.47.201:10000/
```

Should default to localhost.

```suggestion
  LandingPage: http://localhost:10000/
```